src/test/scala/io/iohk/ethereum/vm/OpCodeGasSpec.scala

```diff
@@ -3,15 +3,17 @@ package io.iohk.ethereum.vm
 import org.scalatest.{FunSuite, Matchers}
 import org.scalatest.prop.PropertyChecks
 import Generators._
-import GasFee._
 import UInt256.{One, Two, Zero}
 import io.iohk.ethereum.domain.{Account, Address}
 import io.iohk.ethereum.vm.MockWorldState.PS
 
-
 // scalastyle:off magic.number
```

Comment:
This should not be necessary: see https://github.com/input-output-hk/etc-client/blob/phase/3/txExecution/scalastyle-test-config.xml#L48